app/views/index.blade.php

```diff
@@ -26,7 +26,51 @@
     @endif --}}
 
     <h1>{{ Lang::get('cachet.past_incidents') }}</h1>
+
+    <?php
+        $today = Carbon::now();
+        $startDate = Carbon::now();
+
+        // Check if we have another starting point
+        if (Input::has('start_date')) {
```

Comment:
Yea I know http should't be on the views, this was a proof of concept with what had been done before. This should be all moved to the controller. Ref https://status.github.com/messages